signing/src/main/java/tech/pegasys/web3signer/signing/config/SignerLoader.java

```diff
@@ -143,40 +202,57 @@ private MappedResults<ArtifactSigner> processMetadataFilesInParallel(
+  private MappedResults<ArtifactSigner> mapToArtifactSignerSequentially(
+      final Collection<SigningMetadata> signingMetadataCollection,
+      final SignerParser signerParser) {
+    if (signingMetadataCollection.isEmpty()) {
+      return MappedResults.newSetInstance();
+    }
+
+    try {
+      return mapToArtifactSigner(signingMetadataCollection, signerParser, false);
+    } catch (final Exception e) {
+      return MappedResults.errorResult();
+    }
+  }
+
+  private MappedResults<ArtifactSigner> mapToArtifactSigner(
+      final Collection<SigningMetadata> signingMetadataCollection,
+      final SignerParser signerParser,
+      boolean useParallelStream) {
+    LOG.info("Converting signing metadata to Artifact Signer with parallel: {}", useParallelStream);
 
-    final Instant start = Instant.now();
     final AtomicLong configFilesHandled = new AtomicLong();
     final AtomicInteger errorCount = new AtomicInteger(0);
 
+    final Stream<SigningMetadata> entryStream =
+        useParallelStream
+            ? signingMetadataCollection.parallelStream()
+            : signingMetadataCollection.stream();
+
     final Set<ArtifactSigner> artifactSigners =
-        configFilesContents.entrySet().parallelStream()
+        entryStream
             .flatMap(
                 metadataContent -> {
                   final long filesProcessed = configFilesHandled.incrementAndGet();
                   if (filesProcessed % FILES_PROCESSED_TO_REPORT == 0) {
-                    LOG.info("{} metadata configuration files processed", filesProcessed);
+                    LOG.info("{} signing metadata processed", filesProcessed);
                   }
                   try {
-                    return signerParser.parse(metadataContent.getValue()).stream();
+                    return signerParser.parse(List.of(metadataContent)).stream();
                   } catch (final Exception e) {
-                    renderException(e, metadataContent.getKey().toString());
+                    LOG.error(
+                        "Error converting signing metadata to Artifact Signer: {}", e.getMessage());
+                    LOG.debug(ExceptionUtils.getStackTrace(e));
                     errorCount.incrementAndGet();
-                    return null;
+                    return Stream.empty();
                   }
                 })
-            .filter(Objects::nonNull)
             .collect(Collectors.toSet());
-    final String timeTaken =
-        DurationFormatUtils.formatDurationHMS(Duration.between(start, Instant.now()).toMillis());
-    LOG.info("Total configuration metadata files processed: {}", configFilesHandled.get());
     LOG.info(
-        "Total signers loaded from configuration files: {} in {} with error count {}",
-        artifactSigners.size(),
-        timeTaken,
-        errorCount.get());
+        "Total signing metadata converted to Artifact Signer (parallel={}): {}",
+        useParallelStream,
+        artifactSigners.size());
```

Comment:
Why remove the error count?

Reply:
the error count has been moved to L:95:

`LOG.info(
        "Total Artifact Signer loaded via configuration files: {}\nError count {}\nTime Taken: {}.",
        metadataResult.getValues().size(),
        metadataResult.getErrorCount(),
        calculateTimeTaken(conversionStartInstant));`